draco/analysis/sidereal.py

```diff
@@ -182,8 +183,13 @@ class SiderealRegridder(Regridder):
         Width of the Lanczos interpolation kernel.
     snr_cov: float
         Ratio of signal covariance to noise covariance (used for Wiener filter).
+    down_mix: bool
+        Down mix the visibility prior to interpolation using the fringe rate
+        of a source at zenith.  This is un-done after the interpolation.
     """
 
+    down_mix = config.Property(proptype=bool, default=False)
+
     def setup(self, manager):
         """Set the local observers position.
```

Comment:
I think we want to test in here if the observer is a Telescope or not, and whether down_mix has also been set.

Reply:
Actually, looking a bit more closely at the broader code, it's pretty clear in the SiderealRegridder code that the argument to setup should be a Telescope class (or ProductManager), and so I think it's fine. The place we need to change is really in ch_pipeline I think, so I'm changing this to approve.